chsdi/templates/htmlpopup/zweitwohnungsanteil.mako

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c,lang)">
+<%
+    lang = lang if lang in ('fr','it','rm','en') else 'de'
```

Comment:
I don't think this does much, it already defaults to 'de' at a higher level in the app if the lang is not the available langs.